src/screens/LocationDetailsScreen.tsx

```diff
@@ -1,19 +1,364 @@
+  const [newWaitTime, setNewWaitTime] = useState(0);
+
+  const chartData = [
+    { quarter: 7, earnings: 1 },
+    { quarter: 8, earnings: 2 },
+    { quarter: 9, earnings: 1 },
+    { quarter: 10, earnings: 4 },
+    { quarter: 11, earnings: 6 },
+    { quarter: 12, earnings: 15 },
+    { quarter: 13, earnings: 6 },
+    { quarter: 14, earnings: 4 },
+    { quarter: 15, earnings: 5 },
+    { quarter: 16, earnings: 5 },
+    { quarter: 17, earnings: 7 },
+    { quarter: 18, earnings: 3 },
+    { quarter: 19, earnings: 15 },
+    { quarter: 20, earnings: 3 },
+    { quarter: 21, earnings: 2 },
+    { quarter: 22, earnings: 5 },
+    { quarter: 23, earnings: 1 },
+  ];
+
+  const poiData = {
+    name: "Centro",
+    address: "Commons Building",
+    distance: 2.1,
+    hours: [
+      { "Monday - Friday": "7:00 AM - 11:00 PM" },
+      { "Saturday & Sunday": "10:00 AM - 11:00 PM" },
+    ],
+    lastUpdated: 4,
+    waitTime: 10,
+    currentHour: 14,
+  };
+
+  const submitNewWaitTime = () => {
+    // send new wait time to backend
+    // api call with newWaitTime
+    setSubmittedSuccessModal(true);
+    setTimeout(() => {
+      setSubmittedSuccessModal(false);
+    }, 2000);
+  };
+
+  const confirmNewWaitTime = () => {
+    // send wait time to backend
+    setConfirmSuccessModal(true);
+    setTimeout(() => {
+      setConfirmSuccessModal(false);
+    }, 2000);
+  };
+
+  const onRefresh = useCallback(() => {
+    setRefreshing(true);
+    setTimeout(() => {
+      setRefreshing(false);
+    }, 1000);
+  }, []);
+
+  return (
+    <View style={theme.styles.screenContainer}>
+      <ScrollView
+        style={styles.scrollContainer}
+        refreshControl={
+          <RefreshControl
+            refreshing={refreshing}
+            onRefresh={onRefresh}
+            progressBackgroundColor={theme.refreshIconBackgroundColorAndroid} // Android only: Background color of refresh icon
+            tintColor={theme.iconColor}
+          />
+        }
+      >
+        <View style={styles.imgContainer}>
+          {/* TODO
+              Add height and width once we know a sample image from the backend
+              And change source */}
+          <Image
+            style={styles.img}
+            source={require("@assets/images/centro.png")}
+          />
+        </View>
+        <View style={styles.addressContainer}>
+          <StyledText style={styles.headingText}>{poiData.name}</StyledText>
+          <StyledText style={styles.detailsText}>
+            {poiData.address + " • " + poiData.distance + " km away"}
+          </StyledText>
+        </View>
+        <View style={styles.divider} />
+        <View>
+          <StyledText style={styles.headingText}>Hours</StyledText>
+          {poiData.hours.map((day) => {
+            return (
+              <StyledText key={Object.keys(day)[0]} style={styles.detailsText}>
+                {Object.keys(day) + ", " + Object.values(day)}
+              </StyledText>
```

Comment:
This looks very weird, maybe we can sanitize the output from the backend to just give the strings directly so you don't have to do weird parsing here.

Reply:
It does, I will just keep it like this for now until we have an idea of what the backend will give and how we will sanitize it